lib/plugins/Label_83.test.ts

```diff
@@ -0,0 +1,183 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_83 } from './Label_83';
+
+test('matches Label 83 qualifiers', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_83(decoder);
+
+  expect(decoderPlugin.decode).toBeDefined();
+  expect(decoderPlugin.name).toBe('label-83');
+});
+
+test('decodes Label 83 variant 1', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_83(decoder);
+
+  // https://globe.adsbexchange.com/?icao=A2A3B5&showTrace=2024-09-22&timestamp=1726967032
+  const text = 'KLAX,KEWR,220103, 40.53,- 74.47, 3836,212, 140.0, 19700';
+  const decodeResult = decoderPlugin.decode({ text: text });
+  console.log(JSON.stringify(decodeResult, null, 2));
+
+  expect(decodeResult.decoded).toBe(true);
+  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.name).toBe('label-83');
+  expect(decodeResult.formatted.description).toBe('Airline Defined');
+  expect(decodeResult.message.text).toBe(text);
+  expect(decodeResult.raw.departure_icao).toBe('KLAX');
+  expect(decodeResult.raw.arrival_icao).toBe('KEWR');
+  expect(decodeResult.raw.day_of_month).toBe('22');
+  expect(decodeResult.raw.position.latitude).toBe(40.53);
+  expect(decodeResult.raw.position.longitude).toBe(-74.47);
+  expect(decodeResult.raw.altitude).toBe('3836');
+  expect(decodeResult.raw.groundspeed).toBe('212');
+  expect(decodeResult.raw.heading).toBe('140.0');
+  expect(decodeResult.remaining.text).toBe('19700');
+  expect(decodeResult.formatted.items.length).toBe(6);
+  expect(decodeResult.formatted.items[0].type).toBe('origin');
+  expect(decodeResult.formatted.items[0].value).toBe('KLAX');
+  expect(decodeResult.formatted.items[1].type).toBe('destination');
+  expect(decodeResult.formatted.items[1].value).toBe('KEWR');
+  expect(decodeResult.formatted.items[2].type).toBe('aircraft_position');
+  expect(decodeResult.formatted.items[2].value).toBe('40.530 N, 74.470 W');
+  expect(decodeResult.formatted.items[3].type).toBe('altitude');
+  expect(decodeResult.formatted.items[3].value).toBe('3836 feet');
+  expect(decodeResult.formatted.items[4].type).toBe('aircraft_groundspeed');
+  expect(decodeResult.formatted.items[4].value).toBe('212');
```

Comment:
This is probably my fault, but can we get a unit on the formatted groundspeed? (Knots?)